system/jcstress/playlist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>jcstress_SampleTestBench</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -jar $(Q)${TEST_ROOT}$(D)systemtest_prereqs$(D)jcstress$(D)jcstress-latest.jar$(Q) -jvmArgs $(Q)$(JVM_OPTIONS) -t SampleTestBench$(Q); \
+		$(TEST_STATUS)</command>
+		<versions>
+			<version>11</version>
```

Comment:
Should this be 8+? If so, we do not need the version tag.

Reply:
Good catch. Updated, works. Grinder/23855.